app/views/layouts/base.html.haml

```diff
@@ -22,7 +22,7 @@
   = csrf_meta_tags
 
 %body
-  = render 'shared/nav' unless params[:controller] == "home"
+  = if params[:controller] != "home" then render 'shared/nav' else render 'home/nav'end
```

Comment:
It's more readable if you use don't use the negative. I would make this a multiline if statement though.

```ruby
- if params[:controller] == "home"
  render 'shared/nav'
- else
  render 'home/nav'
```